docker/env/.lido_terra.prod.env

```diff
@@ -13,3 +13,4 @@ BASSET_CONTRACTS_VERSION=2
 SERVICE_NAME=lido_terra_mainnet
 NETWORK_GENERATION=columbus-5
 DELEGATIONS_DISTRIBUTION_CONFIG_NUM_MEDIAN_ABSOLUTE_DEVIATIONS=3
+EXTERNAL_TERRA_BOTS_HOST=23.88.32.40
```

Comment:
i'm not sure we want to put this here. please update the readme with info about the new env var and remove the prod value from this file (just leave blank). update the notion how to if necessary

Reply:
I added it because it won't build without it now.

Reply:
It needs some non blank value

Reply:
okay, but still — what's the point of sharing the actual address of our governance bot?

Reply:
hm, maybe need to put dummy stuff here